### PR TITLE
Describe all fields of SecretRequest messages

### PR DESCRIPTION
**Pull request by pirapira**
Secret Request contains more fields than were described.  I also added a link to the corresponding part of the implementation.

### CODE REVIEW

File: messaging.rst [hunk 1]
```diff
@@ -138,13 +138,25 @@ Invariants
...
+
 +----------------------+---------------+------------------------------------------------------------+
 | Field Name           | Field Type    |  Description                                               |
 +======================+===============+============================================================+
-|  payment_amount      | uint256       | The amount received by the node once secret is revealed    |
+|  cmdid               | one byte      | Value 3 (indicating ``Secret Request``)                    |
++----------------------+---------------+------------------------------------------------------------+
+|  pad                 | three bytes   | Ignored                                                    |
++----------------------+---------------+------------------------------------------------------------+
+|  message identifier  | uint64        | An identifier used in ``Delivered`` acknowledgments        |
```
Comment (LefterisJP) [inline messaging.rst]: Delivered and Processed

Comment (pirapira) [reply messaging.rst]: Will fix in this PR.

**Comment by pirapira** [comment]
Merge?

**Comment by LefterisJP** [comment]
Merge!

**Pull request merged**